src/components/BulkUploaderFilePicker.css

```diff
@@ -0,0 +1,60 @@
+:root {
+  --file-picker--background-color: var(--color--gray--lighter);
+  --file-picker--color: var(--color--gray--darker);
+  --file-picker--border: 1px solid var(--color--gray--medium);
+  --file-picker--max-width: none;
+  --file-picker--button--cursor: pointer;
+}
+
+.file-picker {
+  max-width: var(--file-picker--max-width);
+}
+
+.file-picker.dragging {
+  --file-picker--background-color: var(--color--gray--light);
+  --file-picker--border: 3px dotted var(--color--gray--medium);
+}
+
+.file-picker.uploading {
+  --file-picker--border: 3px dotted var(--color--gray);
+  --file-picker--button--cursor: no-drop;
```

Comment:
Ooh, interesting. So one reason I have it as `no-drop` is that not only is dropping not possible, but clicking to load the file picker is also not possible. But maybe that _is_ too aggressive.

I'm going to change this to `default` for `.uploading`, and reserve `no-drop` for `.uploading.dragging`… even though I've yet to see a browser that actually displays the (CSS-defined) cursor while dragging a file in. (It shows the file ghost.) Still, it's a fine rule to have in case.